doc/source/ray-dashboard.rst

```diff
@@ -0,0 +1,357 @@
+Ray Dashboard
+=============
+Ray's built-in dashboard provides metrics, charts, and other features that help Ray users to understand Ray clusters and libraries.
+
+Through the dashboard, you can
+
+- Understand worker and machine behaviors by providing resource usages and status of them.
+- Visualize the actor relationship and stats.
+- Communicate to your Ray cluster. For example, you can kill your actors, profiling application, or many others.
+- See Tune jobs and trials information in a glance.
+- Detect cluster anomalies and debug them.
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/dashboard/Dashboard-overview.png
+    :align: center
+
+You can go to
+
+- `Machine View <ray-dashboard.html#id1>`_ to see CPU, RAM, and other resources usages of each machine and worker.
+- `Logical View <ray-dashboard.html#id2>`_ to see actor stats and hierarchy.
+- `Ray Config <ray-dashboard.html#id3>`_ to see autoscaler configuration. 
+- `Tune <ray-dashboard.html#id4>`_ to see running Tune job status.
+
```

Comment:
this is a bit redundant with L5-12; I think you can remove this?

Reply:
Agree. I removed them.